pkg/sync/filepath_sync.go

```diff
@@ -28,7 +33,17 @@ func (fs *FilePathSync) Fetch(_ context.Context) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return string(rawFile), nil
+
+	switch fs.FileType {
```

Comment:
`FileType` here is the same as the value for the `--evaluator` flag of `flagd start ... -e json` (if no evaluator is specified, [`json` is used as the default](https://github.com/open-feature/flagd/blob/8c8e155342154b4e1af87e24e68ab71a9446699e/cmd/start.go#L40))